src/controller/actions/resource.js

```diff
@@ -0,0 +1,120 @@
+import api from '../../controller/api';
+
+export const RESOURCE_LOADING = 'RESOURCE_LOADING';
+export const resource_loading = () => ({
```

Comment:
Let's stick to camelCase for all our methods. Consistency in any code base is really important because consistency allows things to be predictable.

Reply:
done